cumulusci/tasks/metadata_etl/encrypt_all_fields.py

```diff
@@ -0,0 +1,207 @@
+from collections import defaultdict
+from datetime import datetime
+
+import os
+import yaml
+import pytest
+
+from cumulusci.tasks.metadata_etl.base import MetadataSingleEntityTransformTask
+from cumulusci.utils.xml.metadata_tree import MetadataElement
+from cumulusci.utils import os_friendly_path
+from cumulusci.utils.xml import metadata_tree
+from cumulusci.tasks.bulkdata.generate_mapping import GenerateMapping
+from cumulusci.core.exceptions import CumulusCIException, TaskOptionsError
+
+
+class EncryptAllFields(MetadataSingleEntityTransformTask):
+
+    entity = "CustomObject"
+
+    default_blocklist_path = "unencryptable.yml"
+
+    encryptable_field_types = [
+        "Email",
+        "Phone",
+        "Url",
+        "Text",
+        "Date",
+        "DateTime",
+        "LongTextArea",
+    ]
+
+    task_options = {
+        "blocklist_path": {
+            "description": "The path to a YAML settings file of Object.Field entities known to be unencrpytable. "
+            "Defaults to unencryptable.yml. Custom entities must be namespace tokenized."
+        },
+        "timeout": {
+            "description": "The max amount of time to wait in seconds. Defaults to 60.",
+        },
+    }
+
+    def _init_options(self, kwargs):
+        self.task_config.options["api_names"] = "dummy"
```

Comment:
Per our discussion, you should probably comment this or fix the bug that caused it.